packages/animations/test/open_container_test.dart

```diff
@@ -580,6 +580,7 @@ void main() {
     final NavigatorState navigator = tester.state(find.byType(Navigator));
     navigator.pop();
     await tester.pump();
+    await tester.pump();
```

Comment:
Why the extra pump?

Reply:
From offline discussion, we were previously looking up the wrong opacity.